ui/src/admin/components/chronograf/OrganizationsTable.js

```diff
@@ -89,13 +98,35 @@ class OrganizationsTable extends Component {
               currentOrganization={currentOrganization}
             />
           )}
+          <Authorized requiredRole={SUPERADMIN_ROLE}>
+            <table className="table v-center superadmin-config">
+              <thead>
+                <tr>
+                  <th style={{width: 70}}>Config</th>
+                  <th />
+                </tr>
+              </thead>
+              <tbody>
+                <tr>
+                  <td style={{width: 70}}>
+                    <SlideToggle
+                      size="xs"
+                      active={superAdminNewUsers}
+                      onToggle={this.handleChangeAuthConfigSuperAdminNewUsers}
```

Comment:
Suggestion to consider:
``` js 
onToggle={this.props.handleChangeAuthConfig('superAdminNewUsers')}
```
delete handleChangeAuthConfigSuperAdminNewUsers function:
and in OrganizationsPage.js replace handleUpdateAuthConfig with: 

```js
handleChangeAuthConfig=(fieldName)=>(updatedValue)=>{ 
  const { 
    actionsConfig: {updateAuthConfigAsync},
    authConfig,
    links,
  } = this.props
  const updatedAuthConfig= {...authConfig, [fieldName]:updatedValue}
  updateAuthConfigAsync(links.config.auth, authConfig, updatedAuthConfig)
}
```

Reply:
good suggestion - thanks!